packages/client-utils/src/sync-tools.js

```diff
@@ -287,3 +287,99 @@ export const waitUntilInvitationReceived = (addr, io, options) => {
     { reusePromise: true, setTimeout, ...resolvedOptions },
   );
 };
+
+/// ////////// Make sure an election held by a given committee (see @agoric/governance) turned out as expected /////////////
+
```

Comment:
```suggestion
/// ////////// Make sure an election held by a given committee   /////////////
/// ////////// (see @agoric/governance) turned out as expected /////////////

```

Reply:
Resolved in https://github.com/Agoric/agoric-sdk/commit/228bf388dad60f5d798eb25fffffb29e04867a83